dotCMS/src/main/java/com/dotmarketing/servlets/ShortyServlet.java

```diff
@@ -322,30 +322,23 @@ private void addImagePath(final int weight,
                             final Optional<FocalPoint> focalPoint,
                             final int cropWidth,
                             final int cropHeight ) {
-      if(isImage) {
-
-          pathBuilder.append("/filter/");
-          pathBuilder.append(weight+height > 0? "Resize,"      : StringPool.BLANK);
-          if(quality>0) {
-            pathBuilder.append("Quality/quality_q/" + quality);
-          }else {
-            pathBuilder.append(jpeg ? "Jpeg/jpeg_q/75"            : StringPool.BLANK);
-            pathBuilder.append(webp ? "WebP/webp_q/75"        : StringPool.BLANK);
-            pathBuilder.append(jpeg && jpegp ? "/jpeg_p/1"        : StringPool.BLANK);
-          }
-          pathBuilder.append(weight > 0? "/resize_w/" + weight : StringPool.BLANK);
-          pathBuilder.append(height > 0? "/resize_h/" + height : StringPool.BLANK);
-          if(focalPoint.isPresent()) {
-              pathBuilder.append("/fp/" + focalPoint.get());
-          }
-          if(cropWidth>0) {
-              pathBuilder.append("/crop_w/" + cropWidth);
-          }
-          if(cropHeight>0) {
-              pathBuilder.append("/crop_h/" + cropHeight);
-          }
-          
-      }
+        if (isImage) {
+
+            if (quality > 0) {
+                pathBuilder.append("/quality_q/" + quality);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid concatenating nonliterals in a StringBuffer/StringBuilder constructor or append().](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4987090)